.github/workflows/ci.yml

```diff
@@ -86,18 +86,22 @@ jobs:
             arch: x64
             trixi_test: threaded_legacy
           - version: '1.9'
+            os: ubuntu-latest
+            arch: x64
+            trixi_test: threaded_legacy
+          - version: '1.10'
             os: macOS-latest
             arch: x64
             trixi_test: mpi
-          - version: '1.9'
+          - version: '1.10'
             os: macOS-latest
             arch: x64
             trixi_test: threaded
-          - version: '1.9'
+          - version: '1.10'
             os: windows-latest
             arch: x64
             trixi_test: mpi
-          - version: '1.9'
+          - version: '1.10'
             os: windows-latest
             arch: x64
             trixi_test: threaded
```

Comment:
Could you please remove the `macos-14` setup below (all 4 lines), since `macos-14` is by now the default, i.e., covered by `macos-latest` above?